ecs-init/cache/cache.go

```diff
@@ -54,10 +56,21 @@ func NewDownloader() *Downloader {
 	sessionInstance, err := session.NewSession()
 	if err != nil {
 		downloader.region = config.DefaultRegionName
```

Comment:
Do we still need this line, what's the behavior if an error was returned?

Reply:
I'm not sure why this line was needed before, so just leave it here. I will remove it for now.